app/templates/base-cy.html

```diff
@@ -16,6 +16,7 @@
 {%- endif -%}
 
 {%- set footer_content = {
+        'newTabWarning': "Os ydych chi'n byw yng Ngogledd Iwerddon, ewch i wefan <a href='https://www.census.gov.uk/ni'>cyfrifiad Gogledd Iwerddon</a> i gymryd rhan",
```

Comment:
Link needs to include 'wefan' (website)...
```suggestion
        'newTabWarning': "Os ydych chi'n byw yng Ngogledd Iwerddon, ewch i <a href='https://www.census.gov.uk/ni'>wefan cyfrifiad Gogledd Iwerddon</a> i gymryd rhan",
```

Reply:
Changed. Not being a welsh speaker certainly proving a disadvantage.